pkg/remote/aws/s3_bucket_notification_supplier.go

```diff
@@ -74,6 +83,8 @@ func (s *S3BucketNotificationSupplier) listBucketNotificationConfiguration(name,
 			},
 		})
 		if err != nil {
+			logrus.Errorf("ERROOORRR %s", reflect.TypeOf(err))
```

Comment:
I think you forgot to remove this one ^^

Reply:
:disappointed: